### DISCUSSION TITLE
Beginner needs Help

### DISCUSSION THREAD

**Discussion by KassCroute** [neos-sdi/adfsmfa#314] [Q&A] [answered]
Hi everyone, especially redhook62 :) 

Redhook j'aurais besoin de ton aide. Je suis étudiant ingénieur cyber en alternance et je souhaite utiliser votre solution. Je manque pas mal de connaissance donc ne m'en veux pas si je te pose des questions bêtes. J'ai parfaitement réussi à installer la solution cependant je n'arrive pas tellement à l'utiliser. J'aimerais enregistrer des yubikey pour mes utilisateurs mais je ne vois pas comment faire il n'y a rien de semblable à cela dans la doc. Est-ce que tu pourrais m'aider sur ce point ? Egalement si ce n'est pas indiscret y a t il un moyen de te contacter plus facilement que sur une Q&A discord ? 

Merci à toi :) !

**Comment by redhook62** [comment, marked as answer]
Hi KassCroute

Pour pouvoir enregistrer les clés de Yubico, pas de soucis, j'en utilise aussi.
Il faut activer le "Provider" biométrique et le paramétrer  voir : [Biometric Provider](https://github.com/neos-sdi/adfsmfa/wiki/08B-MFA%20Biometric%20Provider)

Ensuite ce sont a vos utilisateurs d'enregistrer leur devices biometriques (sauf Security Key avec code pin) pour leur compte.
Vous devez donc, laisser vos utilisateurs s'enregistrer sur MFA.

sinon, envoyez moi votre téléphone sur le mail indiqué dans les codes sources.

regards
